politeiawww/cmd/politeiawwwcli/commands/help.go

```diff
@@ -0,0 +1,41 @@
+package commands
+
+import "fmt"
+
+type HelpCmd struct {
+	Args struct {
+		Topic string `positional-arg-name:"topic" description:"get information about available commands"`
+	} `positional-args:"true"  required:"true"`
+}
+
+func (cmd *HelpCmd) Execute(args []string) error {
+	topic := cmd.Args.Topic
+
+	if topic == "login" {
```

Comment:
Instead of defining the help messages in the `help` command itself, I think it would make more sense to put them in the same file as the command they reference.  For example, `LoginCmdHelpMsg` would be defined in `commands/login.go` as a string literal.  Do you agree?

Reply:
Resolved